cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax-alter-user.asciidoc

```diff
@@ -2,11 +2,15 @@
 [source, cypher]
 -----
 ALTER USER name SET {
-      [PLAINTEXT | ENCRYPTED] PASSWORD password
+[PLAINTEXT \| ENCRYPTED] PASSWORD password
             [[SET PASSWORD] CHANGE [NOT] REQUIRED]
```

Comment:
If we remove the space before `[PLAINTEXT \| ENCRYPTED] PASSWORD password` should we half the space before `            [[SET PASSWORD] CHANGE [NOT] REQUIRED]` as well? (goes for all lines)

Reply:
Afaict the spacing hasn't changed vs 4.3 head, but I'm happy to make it consistent with CREATE USER if that's what is preferred.